c-ares-sys/src/ffi.rs

```diff
@@ -577,6 +582,7 @@ extern "C" {
         naddrttls: *mut ::std::os::raw::c_int,
     ) -> ::std::os::raw::c_int;
 }
+#[cfg(cares1_17)]
```

Comment:
and I think you have previously told me that this should be unnecessary too, on the grounds that anyone who tried to use functions that do not exist because of a too-old c-ares would find that their code failed to compile (or link) anyway?

Reply:
yes